qiita_db/test/test_util.py

```diff
@@ -164,6 +164,34 @@ def test_insert_filepaths(self):
         exp_fp = join(qdb.util.get_db_files_base_dir(), "raw_data",
                       "1_%s" % basename(fp))
         self.assertTrue(exists(exp_fp))
+        self.assertFalse(exists(fp))
+        self.files_to_remove.append(exp_fp)
+
+        # Check that the filepaths have been added to the DB
+        obs = self.conn_handler.execute_fetchall(
+            "SELECT * FROM qiita.filepath WHERE filepath_id=%d" % exp_new_id)
+        exp_fp = "1_%s" % basename(fp)
+        exp = [[exp_new_id, exp_fp, 1, '852952723', 1, 5]]
+        self.assertEqual(obs, exp)
+
+    def test_insert_filepaths_copy(self):
+        fd, fp = mkstemp()
+        close(fd)
+        with open(fp, "w") as f:
+            f.write("\n")
+        self.files_to_remove.append(fp)
+
+        exp_new_id = 1 + self.conn_handler.execute_fetchone(
```

Comment:
We are using bigserials in the DB, i.e. they get autoincremented by the database, so that's why I can predict the id. I added a comment anyways.